NorthstarDLL/mods/modmanager.cpp

```diff
@@ -150,18 +159,20 @@ Mod::Mod(fs::path modDir, char* jsonBuf)
 
 		spdlog::info("'{}' registered ConVar '{}'", Name, convar->Name);
```

Comment:
This (alongside similar things) was something that was discussed in discord, and we agreed that it should be changed, as it might be confusing. (convars are also "registered" for the game, which is also logged)

I'm open to suggestions, but perhaps something like 
```suggestion
		spdlog::info("'{}' contains ConVar '{}'", Name, convar->Name);
```
could work?

Reply:
sounds good, cant really think of a better way to do this